src/ResourceManager/Compute/Commands.Compute/Extension/AzureDiskEncryption/DisableAzureDiskEncryption.cs

```diff
@@ -89,6 +89,22 @@ public class DisableAzureDiskEncryptionCommand : VirtualMachineExtensionBaseCmdl
             HelpMessage = "Disable auto-upgrade of minor version")]
         public SwitchParameter DisableAutoUpgradeMinorVersion { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            Position = 6,
```

Comment:
@vermashi nit: the number of positional parameters in a single set should be capped at four or five. After this, it can be difficult for the user to remember the order in which they should provide the values, so our recommendation would be to remove the positions from the parameters you added (I would also include the `DisableAutoUpgradeMinorVersion` parameter above since switch parameters are only named ☺️ )